### DISCUSSION TITLE
Features without Settings or Tabs

### DISCUSSION THREAD

**Discussion by xyhbhtt** [FreeTubeApp/FreeTube#6040] [General]
So I started to realize that FreeTube has some Features without settings or possibilities to make use of them.

### Audio Tracks
The latest Mental Outlaw video startled me as it opened with a robotic voice speaking german. https://youtu.be/-aeFekbm3Nc
![screenshot2](https://github.com/user-attachments/assets/4a31de1c-9e8c-453d-a641-d34716b1da4b)
I'm aware that YT introduced this feature a good while back for a few handpicked channels. 
I'd really like to see a setting to select a default if the feature is used in a video, eg. eng, ger, .... , or what I want to see original audio track.

### Bookmark
Videos in Freetube can be saved with the "add to Favourites" Button. 
![screenshot](https://github.com/user-attachments/assets/4b414766-df6e-4cc7-8cfd-085d7404e60d)
The problem is that there's no obvious way call the favourited videos, btw. I haven't even found a way to list them at all. 
An additional tab on the side, with the setting to hide it, would be nice.
![screenshot3](https://github.com/user-attachments/assets/c1c0c5d2-473f-4c6d-9f95-1b1a3448f67b)

Is there anything else that's sticking out and should have a setting or a way to access?

**Comment by absidue** [comment]
1. Selecting a custom default audio track already has an open feature request, please check the open and closed issues first before opening new ones in the future. Additionally YouTube only introduced the auto-generated dubbed audio tracks very recently, which is why 0.22.0 doesn't handle them correctly, you are thinking of the dubbed audio tracks created and uploaded by the video uploader, they are two different things.
2. The tab exists and it even has a setting, in fact you already have the setting to hide it turned on: `Hide Playlists` in the Distraction Free settings section.

**Reply by xyhbhtt** [reply]
Alright. 
1. I've searched for audio track with is:open and is:closed under discussions without hits.
2. And the hide playlists must have been unclear from the layout of a previous release. I thought it would it would hide playlists in the up next section.
Sorry for me no being thorough.
